### ISSUE TITLE
Project update 

### ISSUE THREAD

**Issue by robsonvn** [nicolai86/terraform-provider-couchdb#1]
Hey @nicolai86 , I wonder if you still want to continue to provide support to this project. I have done some work getting it working again and I can put a PR or perhaps we might try to put this under the Apache umbrella.

**Comment by nicolai86** [comment]
@robsonvn that sounds like a wonderful idea; I started this as a proof of concept and I'd love to bring this back into todays terraform world.

how about we start with your PR and see what's missing to a fully fledged couchdb 3.0/ 3.1 provider?

**Comment by robsonvn** [comment]
Right, I will have to do some work in order to put the PR up, I disabled the document replication because it will require some change in your go wrapper for CouchDB.

Also, I haven't tested with 3.x at all, and I wanted to hear from you if you want to continue providing support to 1.x

**Comment by nicolai86** [comment]
I think we can move support for 1.x into a branch and keep master for 2.x and 3.x. Let me know if there's anything I can help you with to make landing the changes easier.
